docs/usage/config-overview.md

```diff
@@ -0,0 +1,250 @@
+Passing `extends` through to be part of Repository config has two major consequences:
+
+- It allows repository users to be able to use `ignorePresets` to ignore all or part of the `extends` presets, and
+- Presets defined within `extends` in Global config will take _higher_ precedence that "regular" Global config, because it's resolved later
+
+### Using a centralized config
+
+Using "centralized" configs through Renovate presets is important in order to be able to:
+
+- Save time by not repeating yourself in every repo with the same config, and
+- Being able to change settings across and entire Organization or groups of repositories in one place
+
+Once you've created a centralized preset config, there are multiple ways you can pass it through to repositories:
+
+- Defining it in Global config (either `globalExtends` or `extends`)
+- Using it as your Inherited config, or referencing it from Inherited config using `extends`
+- Ensuring it's referenced in Onboarding config so that it's committed as part of the Repository config
+
+The above possibilities go from least to most transparent when it comes to end users.
+
+Global config may be invisible to developers without log access, meaning they could be confused by any settings you apply - via presets or directly - withing Global config.
+For example they wonder why Renovate is behaving differently to its documented default behavior and may even think it's a bug.
+
+Inherited config is visible to developers (it's within a repository they can see) although it's _implicitly_ applied so without log access and if they're not aware to look for an Inherited config repository then they may again be a little confused as to why default behavior has changed.
+
+Extending presets through Repository config is the most explicit and obvious way to use presets from an end user point of view.
+If they wonder why behavior is a certain way, they can look at the Repository config and trace through all presets it references.
+
+## Mend Renovate App Config
+
+The [Mend Renovate App](https://github.com/apps/renovate) is a popular way to use Renovate on GitHub.com so it's important that any of its non-default behavior is documented here.
+
+Importantly, logs for all Renovate jobs by the Mend Renovate App are available through the [Mend Developer Portal](https://developer.mend.io) so end users can view the logs to see which settings are applied.
+
+### Onboarding behavior
+
+If an Organization is installed with "All repositories" instead of "Selected repositories" then Renovate will default to "Silent" mode (`dryRun=lookup`).
+This has been chosen for two reasons:
+
+- Too often an account or org administrator selects this option without realizing it could cause the onboarding of hundreds of repositories, and
+- By offering this option, it means that org administrators _can_ install into All repositories without worrying about the noise, and then let individual Repository admins decide if/when to start onboarding
```

Comment:
I think this currently can't be done by an repo admin. I was not able todo so for a repo in jenkinsci org.

Reply:
@nabeelsaabna can you check this point? We should allow repo admins to be able to override silent/interactive org defaults

Reply:
We'll treat this as a bug if it's not how it's working